lib/generamba/service/remote_plugin.rb

```diff
@@ -0,0 +1,51 @@
+module Generamba
+  module Service
+    class RemotePlugin
```

Comment:
I see now that it's a generic class for any type of plugin installed from a remote repository. And catalog is a plugin too.
However it'd be great if you'll add comments for new classes. Or maybe a brief description for a pull request with some implementation details.